File: wallendar-app/app/src/main/java/com/td/wallendar/home/balances/ui/BalancesViewImpl.java
package com.td.wallendar.home.balances.ui;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.Nullable;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.td.wallendar.home.balances.BalanceAdapter;

public class BalancesViewImpl extends RecyclerView implements BalancesView {

    public BalancesViewImpl(Context context) {
        this(context, null);
    }

    public BalancesViewImpl(Context context, @Nullable AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public BalancesViewImpl(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void bind(final BalanceAdapter balanceAdapter) {
        setHasFixedSize(true);
        setLayoutManager(new LinearLayoutManager(getContext(), LinearLayoutManager.VERTICAL, false));
        setAdapter(balanceAdapter);
    }

}

File: wallendar-app/app/src/main/java/com/td/wallendar/home/balances/ui/BalancesViewImpl.kt
package com.td.wallendar.home.balances.ui

import android.content.Context
import android.util.AttributeSet
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.td.wallendar.home.balances.BalanceAdapter

class BalancesViewImpl @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0) : RecyclerView(context, attrs, defStyleAttr), BalancesView {
    override fun bind(balanceAdapter: BalanceAdapter) {
        setHasFixedSize(true)
        layoutManager = LinearLayoutManager(context, LinearLayoutManager.VERTICAL, false)
        adapter = balanceAdapter
    }
}